python/pycylon/pycylon/data/aggregates.pxd

```diff
@@ -30,12 +33,57 @@ cdef extern from "../../../../cpp/src/cylon/compute/aggregate_kernels.hpp" names
         CQUANTILE 'cylon::compute::QUANTILE'
         CSTDDEV 'cylon::compute::STDDEV'
 
+    cdef cppclass CBasicOptions "cylon::compute::BasicOptions":
+        CBasicOptions()
+        CBasicOptions(cbool skip_nulls)
+
+    cdef cppclass CVarKernelOptions "cylon::compute::VarKernelOptions":
+        CVarKernelOptions()
+        CVarKernelOptions(int ddof, cbool skip_nulls)
+
+
 cdef extern from "../../../../cpp/src/cylon/compute/aggregates.hpp" namespace "cylon::compute":
+    CStatus Sum(const shared_ptr[CTable] & table, int col_idx, shared_ptr[CTable] & output)
+
+    CStatus Count(const shared_ptr[CTable] & table, int col_idx, shared_ptr[CTable] & output)
+
+    CStatus Min(const shared_ptr[CTable] & table, int col_idx, shared_ptr[CTable] & output)
+
+    CStatus Max(const shared_ptr[CTable] & table, int col_idx, shared_ptr[CTable] & output)
+
+    # Column, Scalar based API
+    # cython does not support overloaded methods in cppclass. https://stackoverflow.com/a/42627030/4116268
+    CStatus SumColumn "Sum"(const shared_ptr[CCylonContext] & ctx, const shared_ptr[CColumn] & values,
```

Comment:
nit: about the method signature, should we put the output field as the last argument. Just a thought.